schemas/orchestrator.yml

```diff
@@ -83,6 +83,19 @@
       description: >
         Type of resource being acted upon.
 
+    - name: resource.parent_type
+      level: extended
+      type: keyword
+      example: DaemonSet
+      description: >
+        Type or kind of the parent resource of the resource being acted upon.
+
+    - name: resource.ip
+      level: extended
+      type: ip
+      description: >
+        IP address of the resource being acted upon.
```

Comment:
> IP address of the resource being acted upon.

Can this description be expanded? Not overly detailed, but an additional sentence or two describing why an event would populate `orchestrator.resource.ip` in addition to `host.ip`?

Reply:
I added a short description that [I think] is better than what I had there before and a longer description with an example in the context of k8s. How does that look now?